src/main/scala/strawman/collection/mutable/Set.scala

```diff
@@ -69,9 +69,7 @@ trait Set[A]
   }
 }
 
-object Set {
-  def apply[A](xs: A*): Set[A] = ???
-}
+object Set extends IterableFactory.Delegate[Set](HashSet)
```

Comment:
What is the most common use case for a `Seq`? Is a `LinearSeq` or an `IndexedSeq` more appropriate? It's hard to pick a good default.